std-bits/table/src/main/java/org/enso/table/aggregations/Concatenate.java

```diff
@@ -1,5 +1,6 @@
 package org.enso.table.aggregations;
 
+import javax.swing.JSeparator;
```

Comment:
don't think this is used

Reply:
Right! I don't even know how this import got here... Thanks for noticing this.